docs/semgrep-secrets/generic-secrets.md

```diff
@@ -0,0 +1,60 @@
+---
+slug: generic-secrets
+title: Scan for generic secrets
+hide_title: true
+description: Use Semgrep to identify generic secrets in your code.
+tags:
+ - Semgrep Assistant
+ - Semgrep Secrets
+---
+
+import PL from '@site/src/components/Placeholder';
+
+# Generic secrets (beta)
+
+:::info
+Generic secrets is in beta. Please contact [Support](/support) to request access to this feature.
+:::
+
+Generic secrets detection scans your code for the inadvertent inclusion of credentials, such as API keys, passwords, and access tokens using rules. It then analyzes the results to eliminate false positives, so you only see high-signal results likely to be true positives.
```

Comment:
How is this different from how our current Secrets product behaves? AFAIK we also use rules to scan for inadvertent credentials. Is the "new bit" the Assistant analysis?

Reply:
@s-santillan WDYT?

> Like Semgrep Secrets, which scans for specific secrets, **generic secrets** detection scans your code for the inadvertent inclusion of credentials, such as API keys, passwords, and access tokens using rules. However, generic secrets detection looks for common keywords, such as auth, key, or passwords, and flags anything nearby that appears to be a secret. It then analyzes the results to eliminate false positives, so you only see high-signal results likely to be true positives.

Reply:
SGTM! Ship it!